modules/nf-neuro/tensor/ratio/main.nf

```diff
@@ -0,0 +1,48 @@
+process TENSOR_RATIO {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.2.sif':
+        'scilus/scilus:latest' }"
+
+    input:
+        tuple val(meta), path(image)
+
+    output:
+        tuple val(meta), path("*_Dxx_to_Dyy.nii.gz")        , emit: tensor_x_to_y
+        tuple val(meta), path("*_Dxx_to_Dzz.nii.gz")        , emit: tensor_x_to_z
```

Comment:
I feel the names `ratio_x_on_y` or `eval_x_on_y` (same for x on z) would convey a better meaning. If you agree, please adjust, here and in the meta.